runner/src/main/java/org/jclouds/cli/runner/Main.java

```diff
@@ -254,7 +254,12 @@ private void run(final CommandProcessorImpl commandProcessor, String[] args, fin
                 if (i > 0) {
                     sb.append(" ");
                 }
-                sb.append(args[i]);
+                if (args[i].contains(" ")) {
+                    // quote arguments with spaces
+                    sb.append("\"").append(args[i]).append("\"");
```

Comment:
This addresses my symptoms but how should this handle quotes?  The felix documentation does not discuss this.